.github/CODEOWNERS

```diff
@@ -1 +1,12 @@
-*       @kubawach @mateuszrzeszutek @signalfx/gdi-instrumentation-java
+* @kubawach @mateuszrzeszutek @signalfx/gdi-instrumentation-java
+
+#####################################################
+#
+# Docs reviewers
+#
+#####################################################
+
+*.md    @kubawach @mateuszrzeszutek @signalfx/gdi-instrumentation-java @signalfx/docs
+*.rst   @kubawach @mateuszrzeszutek @signalfx/gdi-instrumentation-java @signalfx/docs
+docs/   @kubawach @mateuszrzeszutek @signalfx/gdi-instrumentation-java @signalfx/docs
+README* @kubawach @mateuszrzeszutek @signalfx/gdi-instrumentation-java @signalfx/docs
```

Comment:
I haven't noticed that before (and it's not really related to your PR 😅 ), but we should use the GDI approver/maintainer teams instead of individuals:
```suggestion
* @signalfx/gdi-java-maintainers @signalfx/gdi-java-approvers @signalfx/gdi-instrumentation-java

#####################################################
#
# Docs reviewers
#
#####################################################

*.md    @signalfx/gdi-java-maintainers @signalfx/gdi-java-approvers @signalfx/gdi-instrumentation-java @signalfx/docs
*.rst   @signalfx/gdi-java-maintainers @signalfx/gdi-java-approvers @signalfx/gdi-instrumentation-java @signalfx/docs
docs/   @signalfx/gdi-java-maintainers @signalfx/gdi-java-approvers @signalfx/gdi-instrumentation-java @signalfx/docs
README* @signalfx/gdi-java-maintainers @signalfx/gdi-java-approvers @signalfx/gdi-instrumentation-java @signalfx/docs
```